internal/pkg/agent/cmd/inspect.go

```diff
@@ -204,25 +212,11 @@ func printMapStringConfig(mapStr map[string]interface{}, streams *cli.IOStreams)
 }
 
 func printConfig(cfg *config.Config, l *logger.Logger, streams *cli.IOStreams) error {
-	caps, err := capabilities.Load(paths.AgentCapabilitiesPath(), l)
-	if err != nil {
-		return err
-	}
-
 	mapStr, err := cfg.ToMapStr()
 	if err != nil {
 		return err
 	}
-	newCfg, err := caps.Apply(mapStr)
-	if err != nil {
-		return errors.New(err, "failed to apply capabilities")
-	}
-	newMap, ok := newCfg.(map[string]interface{})
-	if !ok {
-		return errors.New("config returned from capabilities has invalid type")
-	}
-
-	return printMapStringConfig(newMap, streams)
+	return printMapStringConfig(mapStr, streams)
```

Comment:
It looks like we're changing how `printConfig` works.  Previously it would apply the capabilities to the configuration but now it doesn't.  I'm ++ to getting rid of the `Apply` API in favor of more explicit APIs but do we need to do something here to preserve the previous behavior of `printConfig`?